package.json

```diff
@@ -37,7 +37,7 @@
     "babel-preset-stage-2": "6.13.0",
```

Comment:
Thanks! But please avoid upgrading packages unless it's absolutely necessary. In this case, I don't want these packages upgraded. Could you revert these changes please?